src/components/select/simple-select/simple-select.test.js

```diff
@@ -0,0 +1,1022 @@
+      <Option text="Purple" value="8" />
+      <Option text="Red" value="9" />
+      <Option text="White" value="10" />
+      <Option text="Yellow" value="11" />
+    </SimpleSelect>
+  );
+};
+
+const SimpleSelectEventsComponent = ({ onChange, ...props }) => {
+  const [state, setState] = React.useState("");
+
+  const setValue = ({ target }) => {
+    setState(target.value.rawValue);
+    if (onChange) {
+      onChange(target);
+    }
+  };
+
+  return (
+    <SimpleSelect
+      label="color"
+      value={state}
+      labelInline
+      onChange={setValue}
+      {...props}
+    >
+      <Option text="Amber" value="1" />
+      <Option text="Black" value="2" />
+      <Option text="Blue" value="3" />
+      <Option text="Brown" value="4" />
+      <Option text="Green" value="5" />
+      <Option text="Orange" value="6" />
+      <Option text="Pink" value="7" />
+      <Option text="Purple" value="8" />
+      <Option text="Red" value="9" />
+      <Option text="White" value="10" />
+      <Option text="Yellow" value="11" />
+    </SimpleSelect>
+  );
+};
+
+context("Tests for Simple Select component", () => {
+  describe("check props for Simple Select component", () => {
+    it.each(testData)(
+      "should render Simple Select label using %s special characters",
+      (labelValue) => {
+        CypressMountWithProviders(<SimpleSelectComponent label={labelValue} />);
+
+        getDataElementByValue("label").should("have.text", labelValue);
+      }
+    );
+
+    it.each(testData)(
+      "should render labelHelp message using %s special characters",
+      (labelHelpValue) => {
+        CypressMountWithProviders(
+          <SimpleSelectComponent labelHelp={labelHelpValue} />
+        );
+
+        helpIcon().trigger("mouseover");
+        tooltipPreview().should("have.text", labelHelpValue);
+      }
+    );
+
+    it.each(testData)(
+      "should render placeholder using %s special characters",
+      (placeholderValue) => {
+        CypressMountWithProviders(
+          <SimpleSelectComponent placeholder={placeholderValue} />
+        );
+
+        selectText().should("have.text", placeholderValue);
+      }
+    );
+
+    it.each([["top"], ["bottom"], ["left"], ["right"]])(
+      "should render the help tooltip in the %s position",
+      (tooltipPositionValue) => {
+        CypressMountWithProviders(
+          <SimpleSelectComponent
+            labelHelp="Help"
+            tooltipPosition={tooltipPositionValue}
+          />
+        );
+
+        helpIcon().trigger("mouseover");
+      }
+    );
+
+    it("should check Simple Select is disabled", () => {
+      CypressMountWithProviders(<SimpleSelectComponent disabled />);
+
+      commonDataElementInputPreview()
+        .should("be.disabled")
+        .and("have.attr", "disabled");
+    });
+
+    it("should render Simple Select as read only", () => {
+      CypressMountWithProviders(<SimpleSelectComponent readOnly />);
+
+      selectText().click();
```

Comment:
why do we need to click into it ?

Reply:
To confirm that when it is clicked the list does not extend, thereby proving the input is read only and cannot be amended.